src/marvin/components/ai_classifier.py

```diff
@@ -106,7 +106,7 @@ class AIEnum(Enum, metaclass=AIEnumMeta):
     """
 
     @classmethod
-    def prompt(
+    def to_prompt(
```

Comment:
Sorry I might have jumped the gun on "to prompt", this isn't casting the object but rather defining an output type, so I think "as_prompt" is more appropriate